server/models/List.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose')
+
+const ListSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    unique: true,
```

Comment:
Not sure that it makes sense for this to be unique, multiple users might want to make a list with the same name

Reply:
I'm assuming we should uniquely identify each object by the _id auto generated by mongodb?

Reply:
👍 Exactly